modules/custom/moj_resources/src/Plugin/rest/resource/CategoryMenuResource.php

```diff
@@ -79,48 +79,48 @@ class CategoryMenuResource extends ResourceBase
 
   protected $currentRequest;
 
-  protected $availableLangs;
+  protected $availableLanguages;
 
   protected $languageManager;
 
-  protected $parameter_category;
+  protected $categoryId;
 
-  protected $parameter_language_tag;
+  protected $languageId;
 
-  protected $parameter_prison;
+  protected $prisonId;
 
   public function __construct(
     array $configuration,
-    $plugin_id,
-    $plugin_definition,
-    array $serializer_formats,
+    $pluginId,
+    $pluginDefinition,
+    array $serializerFormats,
     LoggerInterface $logger,
-    CategoryMenuApiClass $CategoryMenuApiClass,
+    CategoryMenuApiClass $categoryMenuApiClass,
     Request $currentRequest,
     LanguageManager $languageManager
   ) {
-    $this->categoryMenuApiClass = $CategoryMenuApiClass;
+    $this->categoryMenuApiClass = $categoryMenuApiClass;
     $this->currentRequest = $currentRequest;
     $this->languageManager = $languageManager;
-    $this->availableLangs = $this->languageManager->getLanguages();
-    $this->parameter_category = self::setCategory();
-    $this->parameter_language_tag = self::setLanguage();
-    $this->parameter_prison = self::setPrison();
-    self::checkLanguageParameterIsValid();
-    self::checkCategoryIsNumeric();
-    parent::__construct($configuration, $plugin_id, $plugin_definition, $serializer_formats, $logger);
+    $this->availableLanguages = $this->languageManager->getLanguages();
+    $this->categoryId = self::setCategoryId();
+    $this->languageId = self::setLanguageId();
+    $this->prisonId = self::setPrisonId();
+    self::checkLanguageIdIsValid();
+    self::checkCategoryIdIsNumeric();
```

Comment:
Do we want to check if Prison ID is numeric?

Reply:
Done 👍